tileserver/__init__.py

```diff
@@ -128,6 +169,30 @@ def handle_request(self, request):
         coord = request_data.coord
         format = request_data.format
 
+        if self.store and layer_data is not self.layer_config.all_layers:
```

Comment:
Sort of. I'm trying to check for a request that's _not_ for the `all` layer here though. I updated the conditional to be more consistent with how it's done below though. Look better now?